package.json

```diff
@@ -7,7 +7,11 @@
   },
   "scripts": {
     "start": "node server",
-    "start-dev": "concurrently \"nodemon server\" \"cd react-ui && npm start\"",
+    "setup-db": "concurrently \"brew install mongodb\" \"brew tap homebrew/services\" \"brew services start mongodb\" \"node createLocalTestDb.js\" \"mongod\"",
```

Comment:
I think you only need concurrently when you want to run processes in parallel—I'd use `&&` here instead.